ui/packages/components/src/utils/colors.ts

```diff
@@ -0,0 +1,59 @@
+export const rgbToHex = (r: number, g: number, b: number): string =>
```

Comment:
I just pulled this out of the monaco editor utilities so it could be used here without being tightly coupled to the editor code..